### PR TITLE
Build package with node2nix, bump version to 0.8.0

### PR DESCRIPTION
**Pull request by eqyiel**
Using the same kind of setup as the riot-web one.

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -628,7 +627,7 @@ in {
       serviceConfig = {
         PermissionsStartOnly = true;
         ExecStart = "${package}/bin/matrix-appservice-irc -c ${configFile} -f ${registration} -p ${toString cfg.port}";
-        WorkingDirectory = "${package}/lib/matrix-appservice-irc";
+        WorkingDirectory = "${package}/lib/node_modules/matrix-appservice-irc";
```
Comment (eqyiel) [inline default.nix:647]: I'm not sure that this is still necessary.

**Comment by Ralith** [comment]
Has this been tested? I tried to bump to 0.8.0 before but wasn't able to get it to come up, though I forget the details.

**Comment by eqyiel** [comment]
@Ralith it's been tested in as far as I'm using it right now, is there something you have in mind that you'd like me to check out though?

**Comment by Ralith** [comment]
That was the biggie. I'll try it out myself soon to be sure.

**Comment by eqyiel** [comment]
Rebased onto master

**Comment by pacien** [comment]
Any news on this?
[v0.10.1](https://github.com/matrix-org/matrix-appservice-irc/blob/0.10.1/CHANGELOG.md) has been released recently.

**Comment by eqyiel** [comment]
@pacien I don't have time to update the PR right now, but it's on my TODO list :sweat_smile:

**Comment by Ralith** [comment]
Apologies for letting this languish; the incredible fragility of this whole stack always makes me reluctant to mess with a currently-working deployment. Still, the bugfixes are starting to stack up, so an update to v0.10 would be nice.

**Comment by eqyiel** [comment]
@Ralith don't sweat it!  I've updated the PR but I'll try it out for a bit first, I'll let you know how it goes.

**Comment by eqyiel** [comment]
It seems like it's working fine.  For reference, my config is here: https://github.com/eqyiel/dotfiles/blob/9e17bc9f8129b1a8b57796e94224aa0384afb659/nix/.config/nixpkgs/nixops/realms/tomoyo.maher.fyi/config/matrix.nix#L124-L180

Feel free to ping me on freenod or matrix if you have any doubts!

**Comment by pacien** [comment]
The latest PR works, but the bridge itself still seems [unstable](https://github.com/matrix-org/matrix-appservice-irc/issues/540).

**Comment by Ralith** [comment]
Seems to work, thanks!

**Pull request merged**